src/app/features/add-course/add-course-window/add-course-window.component.html

```diff
@@ -0,0 +1,48 @@
+<div class="add-course">
+  <h1 class="add-course__title">New Course</h1>
+
+  <form [formGroup]="courseForm" #registerForm="ngForm" (ngSubmit)="submitCourse" class="add-course__form">
+    <label for="title" class="add-course__label">Title
+      <input
+        type="text"
+        id="title"
+        [(ngModel)]="title"
+        formControlName="title"
+        class="add-course__input"
+        [ngClass]="{ 'is-invalid': f.title.errors }"
+      />
+    </label>
+
+
+    <label for="description" class="add-course__label">Description
+      <textarea
+        type="text"
+        id="description"
+        [(ngModel)]="description"
+        formControlName="description"
+        class="add-course__input add-course__input_textarea"
+        [ngClass]="{ 'is-invalid': f.description.errors }"
+      ></textarea>
+    </label>
+
+    <app-duration [formGroup]="courseForm"></app-duration>
+    <app-date [formGroup]="courseForm"></app-date>
+    <app-authors [formGroup]="courseForm"></app-authors>
+
+    <div class="add-course__actions">
+      <button
+        [disabled]="courseForm.invalid"
+        (click)="submitCourse()"
```

Comment:
You're already have (ngSubmit)="submitCourse". The (click)="submitCourse()" is redundant. Hust add type='submit' to the button instead

Reply:
removed, thanks!